### PR TITLE
EOS-24917: Update QSG with verification info

### PR DESCRIPTION
**Pull request by pavankrishnat**
Problem: Quick-Start-Guide(QSG) page missing with page verification info.

Solution: Added a Testing_Verification_History page and linked it to QSG page.

Signed-off-by: pavankrishnat <pavan.k.thunuguntla@seagate.com>

-----
[View rendered README.md](https://github.com/pavankrishnat/cortx-hare/blob/patch-2/README.md)

### CODE REVIEW

**Comment by mssawant** [comment]
@pavankrishnat, kindly elaborate the commit message, it does not state clearly what was missing and what was added.

File: README.md [hunk 1]
```diff
@@ -300,3 +300,9 @@ Solution: install cortx-py-utils RPM and retry.
 
 * [Hare RFCs](rfc/README.md)
 * [Hare wiki](https://github.com/Seagate/cortx-hare/wiki)
+
+## Tested by:
```
Comment (suvratjoshi) [inline README.md:304]: Is this a requirement to add this particular section?
If it is a requirement, then is it a good idea to maintain it in a separate file named something like "Testing_Verification_History", as not to mix it up with contents of Readme? As this verification needs to be done at the end of every PI, the Readme will keep growing.

**Comment by mssawant** [comment]
@pavankrishnat, why do we need to add a `Tested by` section? isn't it a continuous activity? cc @knekrasov

File: README.md (hunk 1 above)
Comment (pavankrishnat) [reply README.md:304]: Kishore (motr team) suggested to add this as a practice. I am ok to created separate page and link it to README 
https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst

**Comment by pavankrishnat** [comment]
> @pavankrishnat, kindly elaborate the commit message, it does not state clearly what was missing and what was added.

Added

File: README.md (hunk 1 above)
Comment (pavankrishnat) [reply README.md]: Created new Page.

**Comment by pavankrishnat** [comment]
> @pavankrishnat, why do we need to add a `Tested by` section? isn't it a continuous activity? cc @knekrasov

As per [OVA and QSG Release Process](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/95092793/OVA+and+QSG+Release+Process) after every PI QA need to test with GIT_TAG and add GIT_TAG info for records, based on that community team will start verify.

**Review by suvratjoshi — APPROVED** [review]

**Review by mssawant — APPROVED** [review]

**Pull request merged**